t/plugin/ua-restriction.t

```diff
@@ -739,3 +739,51 @@ hello world
     }
 --- response_body
 passed
+
+
+
+=== TEST 33: the element in allowList is null
+--- config
+    location /t {
+        content_by_lua_block {
+            local plugin = require("apisix.plugins.ua-restriction")
+            local conf = {
+                allowlist = {
+                    null
+                },
+            }
+            local ok, err = plugin.check_schema(conf)
+            if not ok then
+                ngx.say(err)
+            end
+
+            ngx.say("done")
+        }
+    }
+--- response_body
+property "allowlist" validation failed: expect array to have at least 1 items
```

Comment:
Look like the conf is not rejected because of the bad regex? Maybe we need a wrong regex to test it.

Reply:
No problem